recipes/_compute_base_config.rb

```diff
@@ -26,6 +26,31 @@
 
 nfs_master = node['cfncluster']['cfn_master']
 
+# EFS
+# Get shared_dir path and mount EFS filesystem
+efs_shared_dir = "/"+node['cfncluster']['cfn_efs_shared_dir'].split(',')[0]
```

Comment:
Missing spaces around the `+`. See [SpaceAroundOperators](https://www.rubydoc.info/gems/rubocop/0.10.0/Rubocop/Cop/Style/SpaceAroundOperators).